### DISCUSSION TITLE
May I know why my website still cannot be accessed by MYNAME.is-a.dev even my .json file has been added to the domains folder?

### DISCUSSION THREAD

**Discussion by Juin28** [is-a-dev/register#14259] [Q&A] [answered]
I have created a pull request to add my .json file into the domains folder in last Friday and it has been approved and added to the folder. May I know why my website still cannot be accessed by MYNAME.is-a.dev link?

**Comment by wdhdev** [comment]
What's your PR #?

**Reply by Juin28** [reply]
14229, whenever I wish to access it, it always show up this error 
![Screenshot 2024-06-03 100304](https://github.com/is-a-dev/register/assets/114983636/d0388340-7f43-4bc5-8f85-ba962e729c1e)

**Reply by wdhdev** [reply, marked as answer]
It seems to load for me, try clearing your browser cache or using another browser.

**Reply by Juin28** [reply]
It works for now, thank you.
